### PR TITLE
{vis}[intel/2018b] Blender v2.79b, OpenImageIO v1.8.16, OpenEXR v2.3.0, ...

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['https://github.com/openexr/openexr/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['8243b7de12b52239fe9235a6aeb4e35ead2247833e4fbc41541774b222717933']
+
+builddependencies = [('CMake', '3.12.1')]
+
+sanity_check_paths = {
```
Comment (ocaisa) [inline easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb]: I find this hard to read, how about
```
sanity_check_paths = {
    'files': ['lib/lib%s-2_3.%s' % (x, SHLIB_EXT) for x in
              ['Half', 'Iex', 'IexMath', 'IlmImf', 'IlmImfUtil', 'IlmThread', 'Imath']] +
             ['bin/exr%s' % x for x in
              ['envmap', 'header', 'makepreview', 'maketiled', 'multipart', 'multiview', 'stdattr']],
    'dirs': ['include/OpenEXR', 'share'],
}
```

Comment (boegel) [reply easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb]: Please also avoid hardcoding version numbers, use `%(version_major)s` and `%(version_minor)s`:

```python
sanity_check_paths = {
    'files': ['lib/lib%s-%%(version_major)s_%%(version_minor)s.%s' % (x, SHLIB_EXT) for x in
              ['Half', 'Iex', 'IexMath', 'IlmImf', 'IlmImfUtil', 'IlmThread', 'Imath']] +
             ['bin/exr%s' % x for x in
              ['envmap', 'header', 'makepreview', 'maketiled', 'multipart', 'multiview', 'stdattr']],
    'dirs': ['include/OpenEXR', 'share'],
}

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 7 (4 easyconfigs in this PR)
node3143.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6390e4e181f5b40ea5158df00a816599 for a full test report.

File: easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb [hunk 2]
```diff
@@ -15,6 +15,8 @@ checksums = ['8243b7de12b52239fe9235a6aeb4e35ead2247833e4fbc41541774b222717933']
 
 builddependencies = [('CMake', '3.12.1')]
 
+dependencies = [('Boost', '1.67.0')]
```
Comment (ocaisa) [inline easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb]: I saw the error that caused this was
```
boost Python is a required dependency when OPENEXR_BUILD_PYTHON_LIBS is set
```
so perhaps we should actually be unsetting `OPENEXR_BUILD_PYTHON_LIBS` or you should be including Python as a dependency.

Comment (boegel) [reply easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb]: @smoors Why not leave `OPENEXR_BUILD_PYTHON_LIBS` enabled and include `Boost.Python` as a dep? Seems like minimal effort...

That would also require the customary `versionsuffix` of course (and including `Python` as a direct dep to ensure `%(pyver)s` template is resolved).

Comment (smoors) [reply easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb]: I could try that, but Boost.Python is version 1.68.0 for 2018b (annoyingly), whereas OpenImageIO and Blender use Boost 1.67.0 (Blender recommends this version).
it works as it is now, and I prefer not to add Python as it is not needed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (4 easyconfigs in this PR)
node2608.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/533ad9858fb96091ce193e3541e426d9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2113.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1278096af894e8c9cebb05688452b54b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**